src/seedu/addressbook/Main.java

```diff
@@ -73,6 +73,7 @@ private void start(String[] launchArgs) {
     /** Prints the Goodbye message and exits. */
     private void exit() {
         ui.showGoodbyeMessage();
+        ui.showTagging(addressBook.getAllTaggings());
```

Comment:
Nice to get this to actually print even though the exercise told you to assume that it'll happen automatically.